lib/razor/config.rb

```diff
@@ -43,7 +43,7 @@ def initialize(env, fname = nil, defaults_file = nil)
       # Use the filename given, or from the environment, or from /etc if it
       # exists, otherwise the one in our root directory...
       fname ||= ENV["RAZOR_CONFIG"] ||
-        (File.file?('/etc/razor/config.yaml') and '/etc/razor/config.yaml') ||
+        (File.file?('/etc/puppetlabs/razor-server/config.yaml') and '/etc/puppetlabs/razor-server/config.yaml') ||
```

Comment:
I think either approach is good, but I'd prefer the invisible-to-user approach if it's not too difficult. Maybe if the config.yaml file is in a different location, or for some reason we cannot move it, then we just instruct the user on how to copy those changes over.